examples/test_examples.py

```diff
@@ -219,10 +166,9 @@ def testExamples(frontend, f):
     import sys
     print("test complete")
     sys.stdout.flush()
-    import time
-    while True:  ## run a little event loop
-        pg.QtGui.QApplication.processEvents()
-        time.sleep(0.01)
+    pg.Qt.QtCore.QTimer.singleShot(1000, pg.Qt.QtWidgets.QApplication.quit)
+    pg.Qt.QtWidgets.QApplication.instance().exec_()
+    # pg.exit()
```

Comment:
remove commented out line, or is there a reason we want to keep it?

Reply:
removed